h2o-py/h2o/model/model_base.py

```diff
@@ -460,16 +460,15 @@ def training_model_metrics(self):
         return self._model_json["output"]["training_metrics"]._metric_json
     
     def model_performance(self, test_data=None, train=False, valid=False, xval=False, auc_type=None, 
-                          auuc_type=None, auuc_nbins=-1):
```

Comment:
This parameter is not able to be used. Default model thresholds are used to calculate AUUC. Change in this function is not allowed.